src/standalone/Snow/snow_parameterizations.jl

```diff
@@ -363,3 +370,202 @@ function energy_from_T_and_swe(S::FT, T::FT, parameters) where {FT}
     end
 
 end
+
+"""
+    compat_density(W_i::FT, sliq::FT, ρ::FT, Δt_t::FT, Tsnow::FT, density::Anderson1976) where {FT}
+Returns the new compressed density ρ_new from the input ρ under the compaction model defined
+by Snow17 for a given `Anderson1976` parameterization configuration.
+"""
+function compat_density(
```

Comment:
what does "compat_density" mean?

Reply:
can change the name to be clearer

Reply:
changed to "compact" instead of "compat"